Makefile

```diff
@@ -3,13 +3,13 @@ include boilerplate/generated-includes.mk
 SHELL := /usr/bin/env bash
 
 # Extend Makefile after here
-CONTAINER_ENGINE?=docker
+CONTAINER_ENGINE=docker
```

Comment:
This is redundant.

Reply:
Worse actually: now it will ignore `CONTAINER_ENGINE` set in the environment.

We should actually just remove this line (though arguably that could be a separate PR) since it's already [set properly and sanely in boilerplate's standard.mk](https://github.com/openshift/boilerplate/blob/f2167d69d378ebb468a49e54f2cae068ab10a383/boilerplate/openshift/golang-osd-operator/standard.mk#L19).